tests/e2e_regression/run.sh

```diff
@@ -103,7 +104,7 @@ for (( i=0; i<nCases; i++ )); do
     >/tmp/pretty 2>/dev/null \
   && cp /tmp/pretty "$outDir/$name.body" || true
   # Sanitize the current timestamp out of the response header so that diffs are stable
-  sed -E -i 's/^(Date|Content-Length|Last-Modified): .*/\1: UNINTERESTING/g' "$outDir/$name.headers"
+  sed -E -i '' 's/^(Date|Content-Length|Last-Modified): .*/\1: UNINTERESTING/g' "$outDir/$name.headers"
```

Comment:
```suggestion
  sed -E -i='' 's/^(Date|Content-Length|Last-Modified): .*/\1: UNINTERESTING/g' "$outDir/$name.headers"
```

Otherwise in linux/bash (really, On My Machine (TM)), `''` is taken to be the command and `s/.../` is taken to be another file path.